pkg/bootstrap/os/tasks.go

```diff
@@ -163,6 +163,7 @@ var (
 		"iptables -X",
 		"iptables -F -t nat",
 		"iptables -X -t nat",
+		"ipvsadm -C",
```

Comment:
`ipvsadm` may not be installed on the node.

Reply:
I understand, so I call it "best-effort"...😥

Or do we just need to show some tips?

Reply:
I think we can handle it this way at present. 
If we add a prompt, we need to check `ipvsadm` first on nodes.

Reply:
> I think we can handle it this way at present. If we add a prompt, we need to check `ipvsadm` first on nodes.

Do you mean we don't need to handle the leftover IPVS rules? (I think that's also reasonable.)

Reply:
Just like this pr, I think we just need to add this command here.
Or we can also help users install `ipvsadm`, if they choose to use kk to install dependent software. 
https://github.com/kubesphere/kubekey/blob/37dd9a43502597fd4afd442ec5efc76d2fb05680/pkg/bootstrap/os/repository/repository_rpm.go#L89
https://github.com/kubesphere/kubekey/blob/37dd9a43502597fd4afd442ec5efc76d2fb05680/pkg/bootstrap/os/repository/repository_deb.go#L81

Reply:
Follow your advice